docker-compose.yml

```diff
@@ -23,8 +23,8 @@ services:
 
   mysql:
     # build:
-    #   context: ./images/mysql
-    image: islandoracollabgroup/isle-mysql:1.1
+    #   context: ../images/isle-mysql
```

Comment:
is this in preparation to maybe move away from submodules?

Reply:
Yes for sure. I'll also need to update documentation to reflect that. My brain dump most likely neglected that detail.